1403.minimum-subsequence-in-non-increasing-order.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * URL of this problem
+ * https://leetcode.com/problems/minimum-subsequence-in-non-increasing-order/
+ */
+
+/**
+ * @param {number[]} nums
+ * @return {number[]}
+ */
+var minSubsequence = function(nums) {
+    
+    const sortNums = nums.sort((a,b) => b - a);
+    const totalSum = sortNums.reduce((a, b) => a + b);
+    
+    let counter = 0;
```

Comment:
This does not seem self-descriptive.
Maybe another name such as "currSubsequenceSum" is better.